onnxruntime-sys/Cargo.toml

```diff
@@ -17,7 +17,7 @@ keywords = ["neuralnetworks", "onnx", "bindings"]
 [dependencies]
 
 [build-dependencies]
-bindgen = { version = "0.59", optional = true }
+bindgen = { version = "0.62", optional = true }
```

Comment:
Current version is 0.69. just making sure that 0.62 here is intentional

Reply:
Good question. I think it was failing for another reason, but now updating to 0.69 works fine -- done. I did not regenerate the x86 and Arm bindings, better to or safer not to mess with what's working?